docs/guides/fragments/flutter/flutterFuncWarning.md

```diff
@@ -0,0 +1,8 @@
+
+<amplify-callout warning>
+
+### The Functions Category is not yet implemented in Flutter Preview.
```

Comment:
I believe all of these guides relate to the API and Auth categories, right? This copy seems wrong. I'd just remove the warning as these guides are still applicable.